woocommerce/wc-admin
# Add a default gateway for the klarna plugin/gateway `klarna-checkout-for-woocommerce`

As part of woocommerce/woocommerce#32258 this adds the default gateway config for Klarna's `klarna-checkout-for-woocommerce` plugin/gateway.

Alongside some modifications to that plugin, this can be used to configure required production config for Klarna. Klarna supports test/prod keys but for simplicity I have just added support for the prod keys depending on whether you're in an EU store or not.

### Detailed test instructions:

1. Checkout my branch: https://github.com/samueljseay/klarna-checkout-for-woocommerce/pull/1
2. Create a zip of the branch and install this plugin to your test site
3. Go to payment setup from the task list on a site where you've opted out of tracking and chosen either an EU country or US country (you could test both cases if you like but thats more related to the klarna plugin PR)
4. You should see the new Klarna payment method available
5. Click "setup" button for Klarna
6. It should see that the plugin is installed, then provide you the option to enter api username and secret.
7. Fill in these fields with any random values and submit the form
8. Klarna should appear in configured gateways list, clicking manage, and opening the right "Api credentials" menu in Klarna settings should show that the settings have been persisted in the plugin.

cc @woocommerce/gezora 

Any review on the PR in my fork would be welcome too obviously, I will then submit the PR to the Klarna repository.

No changelog required.
